test-website/README.md

```diff
@@ -13,3 +13,28 @@ It contains all kind of situations we currently cope with:
 
 - invalid inline CSS
 - invalid rule in a CSS file
+
+
+## How to use locally
+
+Build the docker image
+
+```
+docker build -t test-website .
+```
+
+Start the test website with appropriate environment variables.
+
+```
+docker run -p 8888:80 --rm --name test-website -e SITE_ADDRESS="test-website.local.oviles.info:80, xn--jdacents-v0aqb.local.oviles.info:80" -e STANDARD_HOSTNAME="http:\/\/test-website.local.oviles.info:8888" -e ACCENTUATED_HOSTNAME="http:\/\/jédéacçents.local.oviles.info:8888" test-website
+```
+
+In the example above, the trick is that we have the following DNS records in place : `local.oviles.info A 127.0.0.1` and `*.local.oviles.info CNAME local.oviles.info`, meaning that any request to local.oviles.info or one of its subdomain will resolve to localhost IP 127.0.0.1 ; we use local ports 8080 for HTTP and 8443 for HTTPS.
+
+You can then open https://test-website.local.oviles.info:8888 in your favorite browser and run manual tests on this website (which uses the other one as sub-site for few resources on special domains with special characters).
+
+## Environments variables needed in Docker image
+
+- SITE_ADDRESS: the site address for Caddy operation ; should contain a non-accentuated and an accentuated hostnames for proper testing
+- STANDARD_HOSTNAME: the full hostname to the non-accentuated / standard hostname as seen from the browser (will be used in HTML/JS/... files) ; will be used by `sed` as replacement, so few chars (e.g. `/`) must be escaped
+- ACCENTUATED_HOSTNAME: the full hostname to the accentuated hostname as seen from the browser (will be used in HTML/JS/... files) ; will be used by `sed` as replacement, so few chars (e.g. `/`) must be escaped
```

Comment:
What's the difference with previous one?

Reply:
I've updated the documentation with a clearer table (I hope) and sample values.

Btw, here I used a "personal" domain name, do you mind if we add required records to `openzim.org` (e.g. `local.openzim.org` and `*.local.openzim.org` so that it looks less suspicious / does not depends on something I'm the only one to manage)

Reply:
Thank you it's a lot clearer now.

- Probably wise to refer to them as `netloc` and not `domain` as it's closer to what this is.
- Given there's no need to resolve, I don't think using openzim domain (or yours) makes sense. It creates a deceptive expectation. There are [reserved names](https://en.wikipedia.org/wiki/Special-use_domain_name) for this purpose. I don't think that having localhost-targetting entries in our zone is wise ; at least until it's code-driven

Reply:
> Probably wise to refer to them as netloc and not domain as it's closer to what this is.
Agreed 

> Given there's no need to resolve ...
There is a need to resolve for local tests, this is why I mention they resolve to 127.0.0.1 and why I suggest these special values for local tests

Reply:
it's a developer-only need (you must run it for it to work) and you've covered it in the README. This is superfluous to have it in the zone. We already have entries we don't remember what for so as I said, until we are code-driven (ie. with comments), I don't think it's worth it.

Reply:
OK, I'm fine with it.